### ISSUE TITLE
[BUG] Groups member sorting should be sorting by name

### ISSUE THREAD

**Issue by kpecknold** [rokwire/illinois-app#3569]
In prod 4.3.61
Group members in university-managed groups should sort by last name (A-Z).

**Comment by TodorBachvarov** [comment]
When we load the Group Members we are using pagination and offset, therefore we will need to implement the sorting by Name/Last Name on the backend (Groups BB). I’ll do the client changes after that. @pmarkhennessy   reassign to the backend team please.

**Comment by TodorBachvarov** [comment]
I've created backend issue in groups building block: https://github.com/rokwire/groups-building-block/issues/386. The client changes will be implemented when it's done.

**Comment by vburgett** [comment]
This is still an issue in 6.1.64. @kpecknold

**Comment by mladen-inabyte** [comment]
@vburgett & @kpecknold, Full name (Display name) is used everywhere within the eco system. The same is applied for the Authman functionality and we're unable to change this logic, because the individual names (e.g first name & last name) are not exposed. Also this potential change first should be delivered from IAM (Active Directory), to Authman (for the managed groups),  to Core BB and then to the rest of the building blocks and lastly to the UI at the final stage. 

Keeping this in mind we have to construct appropriate change management strategy and initiate some actions in order to achieve the final goal - more flexible way to sort by first and last name independently. 

CC @pmarkhennessy  @shurwit

**Comment by kpecknold** [comment]
I don't think this is a big issue. It currently sorts by first name. That will be fine.
